.github/labeler-pull-request-triage.yml

```diff
@@ -1,4 +1,17 @@
 dependencies:
-  - vendor/**/*
+  - 'vendor/**/*'
 documentation:
-  - website/**/*
+  - '**/*.md'
+  - 'examples/**/*''
+  - 'website/**/*'
+provider:
+  - '.circleci/**/*'
+  - '.github/**/*'
+  - '.gitignore'
+  - '.go-version'
+  - '*.md'
+  - 'docs/**/*'
+  - 'GNUmakefile'
+  - 'main.go'
+  - 'vsphere/**/*'
+  - 'website/docs/index.html.markdown'
```

Comment:
@tenthirtyam  - in my review, i made some assumptions, could you explain these additions? I suspect this might've broken the GHAs for pull requests

Thanks!

Reply:
Applies existing labels on the pull requests based on the pull request content. It was present prior, but tidied up base in the same in the AWS provider repo structure.

Ryan

Reply:
Oh, it was also line 5 with the extra " ' " 🤦🏻‍♂️

Reply:
Thanks for explaining - i too was using AWS as a reference, I was unsure if it all applied to this repo as well - thanks for doing the leg work - 🤞 this gets our gha squared away for this repo - thanks for your help! 

Brandy

Reply:
Happy to help!

If you or any of the maintainers need anything. Tejitha (HC PM) can put you touch to my VMware email.

Ryan